src/main/docs/guide/modules-testcontainers.adoc

```diff
@@ -161,6 +161,8 @@ test-resources:
 The `network` key is used to declare the network containers use.
 The `network-aliases` can be used to assign host names to the containers.
 
+The `network-mode` key can be used to join an existing network.
```

Comment:
:x: I don't think this documentation is correct. In order to join an existing network, you need to use the `network` property. This is about the network _mode_ (bridge, etc...). I would rephrase the docs.

Reply:
Well, for my use case, I provide the name of an existing bridge network I want my containers to join.
I agree that this is confusing which is why I suggested making a change to the existing `network` functionality.
What do you think?

Reply:
But I think it's more confusing as it's not aligned with the testcontainers terminology. I think this should be rephrased to:

> The `network-mode` can be used to set the network mode (e.g. including 'host', 'bridge', 'none' or the name of an existing named network.)

Which is basically copied from the `withNetworkMode` javadocs.